scripts/deploy-k8s.sh

```diff
@@ -25,4 +25,6 @@ gcloud config set project emblica-212815
 gcloud config set compute/region europe-north1
 gcloud container clusters get-credentials emblica-cluster-1 --region europe-north1
 
-kubectl set image deployment/${K8S_DEPLOYMENT_NAME} ${K8S_DEPLOYMENT_NAME}=${DOCKER_IMAGE}:${TRAVIS_BUILD_NUMBER} -n rekrysofta
\ No newline at end of file
+kubectl set image deployment/${K8S_DEPLOYMENT_NAME} ${K8S_DEPLOYMENT_NAME}=${DOCKER_IMAGE}:${TRAVIS_BUILD_NUMBER} -n rekrysofta
+
+kubectl exec -it -n rekrysofta $(kubectl get pods -n rekrysofta | grep server | awk '{ print $1 }') -- bash -c 'npm run migrate:production'
```

Comment:
Not the best idea as set image command starts killing the container and recreating a new one with new image. You could end up in situation where migrations are run in the old container using old migrations. If migrations don't have any effect if run multiple times the best way for now might be to run migrations on startup as part of npm start or something

Reply:
Ah, yes, I didn't think of that. Thanks for the suggestion!